wayland-display-core/src/comp/rendering.rs

```diff
@@ -107,15 +120,15 @@ impl State {
                     buffer,
                     self.video_info.as_ref().unwrap(),
                 )
-                .unwrap();
+                    .unwrap();
                 let plane_data = vframe.plane_data_mut(0).unwrap();
                 plane_data.clone_from_slice(map);
             }
 
             buffer
         };
         self.renderer.unbind().map_err(DTRError::Rendering)?;
-
-        Ok((buffer, damage, render_element_states))
+        todo!("Get damage!");
```

Comment:
Previously `damage` was returned by the `render_output` method, how can I get that here?

Reply:
`render_output` returns a `RenderOutputResult` now, which contains both the `damage` and the `render_elements_states` now: https://smithay.github.io/smithay/smithay/backend/renderer/damage/struct.RenderOutputResult.html

It *also* contains a sync-point now, which we would need to wait on before giving a hardware buffer to gstreamer or we might not be done writing to it (or even better give that and the dmabuf to gstreamer, but that api change will take a another few years probably...).
But because we copy to memory anyway, OpenGL should already synchronize for us and we can just ignore it.